src/storage/src/hummock/compactor/compaction_utils.rs

```diff
@@ -183,14 +183,14 @@ fn generate_splits_fast(
         indexes.push(
             FullKey {
                 user_key: FullKey::decode(&key_range.left).user_key,
-                epoch: HummockEpoch::MAX,
+                epoch_with_gap: EpochWithGap::new_from_epoch(HummockEpoch::MAX),
```

Comment:
May want to mask some lower bit as this will break the sanity check in `new_from_epoch`.